middleware/static/index.html

```
@@ -0,0 +1,324 @@
+          </div>
+        </div>
+        <div class="toggle-row">
+          <span class="toggle-label">Tag Writeback</span>
+          <div class="toggle" id="cfg-writeback" onclick="this.classList.toggle('on')"></div>
+        </div>
+        <div class="toggle-row">
+          <span class="toggle-label">Publish Lane Data (Slicer Integration)</span>
+          <div class="toggle" id="cfg-lanedata" onclick="this.classList.toggle('on')"></div>
+        </div>
+        <br>
+        <button class="btn btn-red" onclick="saveConfig()">Save Config & Restart</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)